src/mono/browser/runtime/runtime.c

```diff
@@ -356,3 +357,25 @@ mono_wasm_assembly_find_method (MonoClass *klass, const char *name, int argument
 	MONO_EXIT_GC_UNSAFE;
 	return result;
 }
+
+/*
+ * mono_wasm_marshal_get_managed_wrapper:
+ * Creates a wrapper for a function pointer to a method marked with
+ * UnamangedCallersOnlyAttribute.
+ * This wrapper ensures that the interpreter initializes the pointers.
+ */
+void
+mono_wasm_marshal_get_managed_wrapper (const char* assemblyName, const char* nspace, const char* typeName, const char* methodName, int num_params)
+{
+	MonoError error;
+	mono_error_init (&error);
+    MonoAssembly* assembly = mono_wasm_assembly_load (assemblyName);
```

Comment:
This would force loading all assemblies with any [UCO], which means we also need to make them loaded before runtime start in https://github.com/dotnet/runtime/pull/100141, right ? 

Or could we have this UCO initialization postponed until user code executes ?

cc @maraf

Reply:
This will only be called when the uco method is called and the wrapper func has not been initialized